qiita_pet/templates/show_analyses.html

```diff
@@ -21,6 +22,8 @@
   <table class="table">
     <tr>
       <th>Analysis Name</th>
+      <th>Mapping File</th>
```

Comment:
Can you move the mapping file and the biom file columns to be at the end, before the delete button? Or at least after the analysis description columns, it makes a better organization.

Ideally, I think we should put this columns in an "expand" as it is done for the processed data in the study list, so we can reduce how crowded this page looks. What does others think? cc @biocore/qiita-dev